src/data/editor-links.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
This file is hand-written. The `iana-links.json` file is auto generated from an official source.

if a definition appears in both, this one wins.